about.html

```diff
@@ -0,0 +1,168 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Capstone Module1</title>
+    <link rel="stylesheet" href="about.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link rel="icon" href="./Images/logo-no-background.svg" type="image/svg" />
+  </head>
+  <body>
+    <aside class="aside-section">
+      <div class="top-elements">
+        <i class="fa-brands fa-facebook-f aside-icon"></i>
+        <i class="fa-brands fa-twitter aside-icon"></i>
+        <a href="#">English</a>
+        <a href="#">Page</a>
+        <a href="#">Logout</a>
+      </div>
+    </aside>
+    <section class="nav-bar">
+      <div id="ham-menu">
+        <img
+          onclick="clickX()"
+          id="x"
+          src="https://img.icons8.com/sf-regular/48/FFFFFF/multiply.png"
+        />
+        <ul class="ham">
+          <li>
+            <a href="index.html" class="ham-items" onclick="clickX()">Home</a>
+          </li>
+          <li>
+            <a href="about.html" class="ham-items" onclick="clickX()">About</a>
+          </li>
+          <li>
+            <a href="#contact" class="ham-items" onclick="clickX()">Program</a>
+          </li>
+          <li>
+            <a href="#contact" class="ham-items" onclick="clickX()">Join</a>
+          </li>
+          <li>
+            <a href="#contact" class="ham-items" onclick="clickX()">Sponsor</a>
+          </li>
+        </ul>
+      </div>
+      <header class="head flex-row">
+        <nav>
+          <img
+            onclick="clickMenu()"
+            id="menu"
+            src="./Images/Icon - Menu.png"
+            alt="Menu Logo"
+            class="menu-logo"
+          />
+          <a href="">
+            <img class="logo" src="Images/logo-black.png" alt="logo" />
+          </a>
+          <ul class="menu_items flex-row">
+            <li class="nav_items"><a href="index.html">Home</a></li>
+            <li class="nav_items"><a href="about.html">About</a></li>
+            <li class="nav_items"><a href="">Program</a></li>
+            <li class="nav_items"><a href="">Join</a></li>
+            <li class="nav_items"><a href="">Sponsor</a></li>
+          </ul>
+        </nav>
+      </header>
+      <div class="main-headline">
+        <h3 class="welcome">"JOIN US"</h3>
+        <h3 class="welcome-2">Take A Step</h3>
+        <h1 class="Meco">MECO</h1>
+        <p class="about_des">
```

Comment:
- [ ] **[OPTIONAL]** Kindly, consider using **only** (or mostly) the color provided on the provided design. Your current project has a lot of different colors which makes the project not consistent ([more info about the color template here](https://www.flux-academy.com/blog/how-to-strategically-use-color-in-website-design#:~:text=To%20start%2C%20I%20recommend%20choosing,and%2010%25%20the%20accent%20color.)). In this particular exercises you need to match the provided design colors.

**Your project**
![image](https://user-images.githubusercontent.com/74411135/233724796-557484d2-3653-4e19-9f6c-9f3c21d4f74d.png)

**provided design**
![image](https://user-images.githubusercontent.com/74411135/233724976-20e4ae61-eff5-4c2b-a8cf-93f4ad2016f4.png)

That will make your project look way better and impress the users as the provided design is done by a professional designer 😮 😄 !

**Note:** This project requirement was missed by the previous reviewer so please consider implementing it.